### PR TITLE
Add proper testing + fix some issues found

### PR DESCRIPTION
**Pull request by nvnieuwk**
This PR makes the tests more modular so it's easier to test new features and changes

=> also fixes some small issues found in the meantime

### CODE REVIEW

File: assets/header.txt [hunk 1]
```diff
@@ -34,6 +34,7 @@
 ##INFO=<ID=CIPOS,Number=2,Type=Integer,Description="PE confidence interval around POS">
 ##INFO=<ID=CNV_RATIO,Number=1,Type=Float,Description="Copy number ratio">
 ##INFO=<ID=CHR2,Number=1,Type=String,Description="Chromosome for POS2 coordinate in case of an inter-chromosomal translocation">
+##INFO=<ID=CSQ,Number=1,Type=String,Description="The consequence of the structural variant">
```
Comment (mvheetve) [inline assets/header.txt:37]: maybe also mention this comes from ensembl vep

Comment (nvnieuwk) [reply assets/header.txt]: Done! :p

File: nextflow.config [hunk 2]
```diff
@@ -262,7 +262,7 @@ manifest {
     homePage        = 'https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-structural'
     description     = 'A nextflow pipeline for calling structural variants'
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=22.10.5'
+    nextflowVersion = '!>=23.10.0'
```
Comment (mvheetve) [inline nextflow.config:265]: Does this mean the pipeline doesn't take NF versions more recent than thins?

**Review by mvheetve — APPROVED** [review]
Ok, good to go.

**Pull request merged**